services/cd-service/pkg/cmd/server.go

```diff
@@ -143,6 +143,8 @@ func RunServer() {
 
 		span.Finish()
 
+		go repository.SendRegularlyDatadogMetrics(repo, 3600, repository.GetRepositoryStateAndUpdateMetrics)
```

Comment:
3600 seconds sounds like a lot. I'd start with 5 minutes ( 300 )

Reply:
Okay! Will change it